Glimmer/Assets/Scripts/DetectionAura.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DetectionAura : MonoBehaviour
+{
+    private Light light;
+    private SphereCollider playerCollider;
+    private PlayerMovementController movementController;
+
+    void Start()
+    {
+        this.light = gameObject.GetComponent<Light>();
+        this.playerCollider = gameObject.GetComponent<SphereCollider>();
+        this.movementController = gameObject.GetComponentInParent<PlayerMovementController>();
+    }
+
+    private void Update()
+    {
+        if (this.movementController != null)
+        {
+            Vector3 movementVector = this.movementController.GetMovementVector();
+
+            if (Input.GetKey(KeyCode.LeftShift))
+            {
+                this.CrouchAura(movementVector.magnitude);
+            }
+            else if(Input.GetKey(KeyCode.Space))
+            {
+                this.RunAura(movementVector.magnitude);
+            }
+            else
+            {
+                this.UpdateAura(movementVector.magnitude);
+            }
+        }
+    }
+
+    public void RunAura(float movementMagnitude)
```

Comment:
Again, your methods must be named as a verb, not as a noun... please rename this, as well as CrouchAura. UpdateAura seems like a proper method name, but is generic